proxy/nginx/lualib/router.lua

```diff
@@ -18,9 +18,10 @@ local redis_port = os.getenv("CERYX_REDIS_PORT")
 if not redis_port then redis_port = 6379 end
 local res, err = red:connect(redis_host, redis_port)
 
--- Return if could not connect to Redis
+-- Exit if could not connect to Redis
 if not res then
-    return
+    ngx.log(ngx.ERR, "failed to connect to redis: " .. err)
+    ngx.exit(ngx.HTTP_INTERNAL_SERVER_ERROR)
```

Comment:
I don't think that this is a really informative exit code, maybe we could document those and add something like `1` here as a start?

Reply:
Actually simple return will produce error of inability to resolve "fallback" host, because container_url is set to that value by default.

That actually will be handled by @fallbacvk location.

In my copy I do "return ngx.exit(ngx.HTTP_BAD_GATEWAY)".

That forces nginx to @fallback route without trying passing to invalid proxy.